### DISCUSSION TITLE
[Projects Beta] Include column for parent issues

### DISCUSSION THREAD

**Discussion by chriskuech** [community/community#11613] [Projects and Issues]
GitHub Issues has a nice and unique workflow for easily making Issues with checklists, then converting checklist items into child issues.  Unfortunately, it is very hard to track these relationships once they are created, or even know whether an issue is a root (parentless) issue or child linked issue.

I would like a column "Parent Issue" that includes a link from the issue in the row to its Parent issue.  If that is too hard to implement, I would at least like a column "Linked Issues" that includes all links to that issue.

**Comment by kajh** [comment]
Any hope for getting something like this?

A similar feature is described on https://github.com/community/community/discussions/7813#discussioncomment-1987387
